treeherder/model/fixtures/repository.json

```diff
@@ -1036,7 +1036,7 @@
         "name": "example-addon",
         "url": "https://github.com/mozilla/example-addon-repo",
         "branch": "master",
-        "active_status": "active",
+        "active_status": "onhold",
```

Comment:
This was also recently added by @davehunt - I think it may need to be left enabled.